pkg/services/searchV2/bluge.go

```diff
@@ -259,6 +259,8 @@ func doSearchQuery(ctx context.Context, logger log.Logger, reader *bluge.Reader,
 		}
 	}
 
+	q.Query = strings.ToLower(q.Query)
```

Comment:
I am torn on what to do here.  I think this is a great™️ quick fix -- the real solution is for us to better understand the analysis chain and fix there.  Tuning our search performance will be a process of understanding how we build the inverted index, and how our queries are constructed to get the best results.

In this particular case I think we *may* want to just lowercase the prefix query:
```
AddShould(bluge.NewPrefixQuery(strings.ToLower(q.Query)).SetField("name").SetBoost(1))
```

BUT -- even better solution is to add an NGram (or [EdgeNGram](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-edgengram-tokenfilter.html)) field so we get fast prefix lookups